CHANGELOG.md

```diff
@@ -1,8 +1,79 @@
-### next version
-* Improved src checking to allow `<img/>` tags (not `<Image/>` components) to
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+
+## [Unreleased]
```

Comment:
The tooling I have added looks for this line in particular. Once it has found it, some extra lines are added and a release date is added as well. The result is then something like this:

```
The format is base on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),

## [Unreleased]



## 0.0.13 - 2019-09-02
### Fixed
- Resizing an image that was smaller than any of the given sizes would fail
```

Reply:
I never had to keep changelog before, I don't mind to start using any tooling because I know it's been messy enough already.

Reply:
Yeah. The idea here is that when you do anything that you think should go in the changelog, you can always put it under the `## [Unreleased]` item. Then, when you run `yarn release`, the version is updated for you in both the package and the changelog. It's a small thing, but in the projects I have used it for, it has been nice to let the computer do that version bumping for me.